packages/@aws-cdk/aws-batch/lib/compute-environment.ts

```diff
@@ -172,7 +172,7 @@ export interface ComputeResources {
    * The minimum number of EC2 vCPUs that an environment should maintain (even if the compute environment state is DISABLED).
    * Each vCPU is equivalent to 1,024 CPU shares. You must specify at least one vCPU.
    *
```

Comment:
"You must specify at least one vCPU." but default is 0?

Reply:
Damn. Nice catch. I'll fix the docstring.